packages/babel-core/test/api.js

```diff
@@ -608,25 +596,25 @@ describe("api", function() {
 
     it("all", function() {
       const script = babel.buildExternalHelpers();
-      assert.ok(script.indexOf("classCallCheck") >= -1);
-      assert.ok(script.indexOf("inherits") >= 0);
+      expect(script.indexOf("classCallCheck") >= -1).toBeTruthy();
```

Comment:
you could use `arrayContaining` matcher here

Reply:
I am not sure if we can use it here, since `typeof script` returns `string`.

Reply:
You can if you use `.split(/\b/)` first :trollface: 

Or use the `stringContaining` matcher 🤷‍♀️ 

Replacing `indexOf`s would need to be a case-by-case thing, not really something a codemod can do without type information 😞

Reply:
Agreed. Thanks @Kovensky for the suggestion. 
Now, changed to using `stringContaining`. :-)